components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/prod/yaml_outs/eamxx_output.decadal.1dailyAVG_native.yaml

```diff
@@ -2,15 +2,15 @@
 ---
 filename_prefix: eamxx_output.decadal.1dailyAVG_native.h
 iotype: pnetcdf
-Averaging Type: Average
+averaging_type: Average
 Max Snapshots Per File: 1
```

Comment:
We can change a few more:

- `Fields` -> `fields`
- `Field Names` - > `Field Names`
- `Max Snapshots Per File` -> `max_snapshots_per_file`
- `Restart` -> `restart`
- `Floating Point Precision` -> `floating_point_precision`

If you want, I can take care of renaming all the IO ones. There are some subtleties (e.g., `Physics PG2` is the name of a grid we create in eamxx, so if we turn it into `physics_pg2`, we also need to change all grids names...which is fine, tbh).

Reply:
just noting I think you meant `Field Names` -> `field_names`.

Reply:
yes, fixed.

Reply:
@jgfouca the more I think about it, the more I think we should also change `Physics GLL`->`physics_gll`, `Physics PG2`->`physics_pg2`, `Dynamics`->`dynamics`. People may be confused by the fact that the grid names in the output yaml files do not follow the convention, and since these names are used as the key for the `field_names` sublists, they should prob abide to the new format convention.

This mod will somehwat heavily impact the source code too, as we do use the grids names quite a bit. I can help with this part, if you want me to.